mila_datamodules/cli/prepare_torchvision.py

```diff
@@ -146,56 +256,286 @@ def __call__(
+        cluster: Compose([
+            SymlinkDatasetFiles(
+                source=f"{datasets_folder}/cityscapes",
+            ),
+            PrepareVisionDataset(tvd.Cityscapes)
+        ])
+        for cluster, datasets_folder in standardized_torchvision_datasets_dirs.items()
+    },
+    tvd.CocoCaptions: {
+        cluster: Compose([
+            SymlinkDatasetFiles(
+                source=f"{datasets_folder}/coco/2017",
+            ),
+            ExtractArchives(
+                archives=[
+                    ("test2017.zip", "."),
+                    ("train2017.zip", "."),
+                    ("val2017.zip", "."),
+                    ("annotations/annotations_trainval2017.zip", "."),
+                    ("annotations/image_info_test2017.zip", "."),
+                    ("annotations/panoptic_annotations_trainval2017.zip", "."),
+                    ("annotations/stuff_annotations_trainval2017.zip", "."),
+                ]
+            ),
+            PrepareVisionDataset(tvd.CocoCaptions)
+        ])
+        for cluster, datasets_folder in standardized_torchvision_datasets_dirs.items()
+    },
+    tvd.CocoDetection: {
+        cluster: Compose([
+            SymlinkDatasetFiles(
+                source=f"{datasets_folder}/coco/2017",
+            ),
+            ExtractArchives(
+                archives=[
+                    ("test2017.zip", "."),
+                    ("train2017.zip", "."),
+                    ("val2017.zip", "."),
+                    ("annotations/annotations_trainval2017.zip", "."),
+                    ("annotations/image_info_test2017.zip", "."),
+                    ("annotations/panoptic_annotations_trainval2017.zip", "."),
+                    ("annotations/stuff_annotations_trainval2017.zip", "."),
+                ]
+            ),
+            PrepareVisionDataset(tvd.CocoDetection)
+        ])
+        for cluster, datasets_folder in standardized_torchvision_datasets_dirs.items()
+    },
+    tvd.FashionMNIST: {
+        cluster: Compose([
+            SymlinkDatasetFiles(
+                source=f"{datasets_folder}/fashionmnist",
+            ),
+            MoveFiles(
+                # Torchvision will look into a FashionMNIST/raw directory to
+                # preprocess the dataset
+                files=[("*", "FashionMNIST/raw/*")]
+            ),
+            PrepareVisionDataset(tvd.FashionMNIST)
+        ])
+        for cluster, datasets_folder in standardized_torchvision_datasets_dirs.items()
+    },
+    tvd.INaturalist: {
+        cluster: Compose([
+            SymlinkDatasetFiles(
+                source=f"{datasets_folder}/inat",
+            ),
+            MoveFiles(
+                # Torchvision will look for those files to preprocess the
+                # dataset
+                files=[
+                    ("train.tar.gz", "2021_train.tgz"),
+                    ("train_mini.tar.gz", "2021_train_mini.tgz"),
+                    ("val.tar.gz", "2021_valid.tgz"),
```

Comment:
Just out of curiosity, are those exactly the files torchvision expects? (The checksums for these files will match those expected from torchvision?)

Reply:
Yes and I'm not sure why `torchvision` decided to rename their files actually